packages/subgraph/src/mappings/GardensTemplate.ts

```diff
@@ -17,10 +17,14 @@ export function handleDeployDao(event: DeployDaoEvent): void {
 }
 
 export function handleSetupDao(event: SetupDaoEvent): void {
-  const org = loadOrCreateOrg(event.params.dao)
+  const daoAddress = event.params.dao
```

Comment:
Could we update to `orgAdddress` for consistency?
```suggestion
  const orgAddress = event.params.dao
```